docs/blog/2024-01-16-binomial-tails.md

```diff
@@ -0,0 +1,390 @@
+---
+title: "Tales from the Binomial Tail: Confidence intervals for balanced accuracy"
+description: How we assess and report detector accuracy
+slug: confidence-intervals-for-balanced-accuracy
+authors:
+  - name: Ted Sandler
+    title: Senior Applied Scientist
+    image_url: https://a-us.storyblok.com/f/1015187/1000x1000/efc35da152/sandlert.jpg
+  - name: Leo Dirac
+    title: CTO and Co-founder at Groundlight
+    image_url: https://a-us.storyblok.com/f/1015187/284x281/602a9c95c5/diracl.png
+tags: [metrics, math, deep dive]
+image: ./images/binomial-tails/balanced-accuracy-bound.png
```

Comment:
I checked that this seems to be rendering properly.  (Once we fix the base url.)

This renders the `og:image` tag as:

```
<meta property="og:image" content="https://www.groundlight.ai/python-sdk/assets/images/balanced-accuracy-bound-ac2f3d3c88bba854b33a96e6372342e9.png" data-rh="true">
```

And `/python-sdk/assets/images/balanced-accuracy-bound-ac2f3d3c88bba854b33a96e6372342e9.png` works on my dev server.

Reply:
I think figure 6 would actually be more representative of the blog's contents.

Reply:
done